wg-automl/README.md

```diff
@@ -0,0 +1,37 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://github.com/kubeflow/community/generator/README.md
+--->
+# AutoML Working Group
+
+WG AutoML is responsible for all aspects of Automated Machine Learning technologies on Kubeflow.
+
+The [charter](charter.md) defines the scope and governance of the AutoML Working Group.
```

Comment:
Do we need to link to the `charter.md` that doesn't exits?

Reply:
> I mis-understood the charter part. I'm adding them back under wg-*/charter.md
Sorry I just can't seem to concentrate today :( 
Fixing it.

Reply:
It's totally fine, thank you for doing this!